src/main/java/Bulldozer.java

```diff
@@ -0,0 +1,12 @@
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("It works");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("It is stopped");
+    }
+
```

Comment:
```suggestion
    @Override
    public void doWork() {
        System.out.println("Bulldozer works");
    }

    @Override
    public void stopWork() {
        System.out.println("Bulldozer stopped");
    }
```

same for other classes